wuzz.go

```diff
@@ -1016,7 +1055,19 @@ func main() {
 
 	initApp(app, g)
 
-	err = app.ParseArgs(g)
+	// load config (must be done *before* app.ParseArgs, as arguments
+	// should be able to override config values). An empty string passed
+	// to LoadConfig results in LoadConfig loading the default config
+	// location. If there is no config, the values in
+	// config.DefaultConfig will be used.
+	app.LoadConfig(configPath)
```

Comment:
Debug view would be great in general, but in this case, we can exit on error and simply print the message imho